bluebottle/analytics/utils.py

```diff
@@ -0,0 +1,32 @@
+from importlib import import_module
+
+from django.conf import settings
+
+
+# NOTE: Stubbed method for testing
+def queue_analytics_record(timestamp, tags={}, fields={}):
+    # Get the default backend for analytics
+    def get_handler_class(handler): 
```

Comment:
It isn't necessary now but I should be looping over all possible backends as there can be multiple. This will be added later and then I think it is neater to have an inline method.

I should really move it into the utils app as something similar is being used in other parts of the code to get a class from a dotted string.